src/complate/index.js

```diff
@@ -4,15 +4,15 @@ import vm from "vm";
 
 export default class Bundle extends _Bundle {
 	async renderString(code, filename, context) {
-		let { renderToString } = await load("rodunj/src/render", "complate extension");
+		let JSXRuntime = await load("complate-ast/dist/runtime", "complate-ast runtime");
 
 		// TODO: generate unique file name to avoid potential race condition for
 		//       concurrent access with identical sources?
 		let id = await this.virtualModule(filename, code);
 		code = await this.generate(id);
 
-		let sandbox = context ? { ...context } : {};
-		let segments = vm.runInNewContext(code, sandbox);
-		return renderToString(...segments);
+		let sandbox = { ...context, JSXRuntime: JSXRuntime };
```

Comment:
complate-ast can generate the import too. Do we want that? If so, it'd need to be configured in `bundling.js`.